internal/pkg/cli/flag.go

```diff
@@ -249,6 +250,7 @@ const (
 	yesFlagDescription          = "Skips confirmation prompt."
 	resourceTagsFlagDescription = `Optional. Labels with a key and value separated by commas.
 Allows you to categorize resources.`
+	diffFlagDescription = "Show the diff of the to-be-deployed CFN template against the deployed one."
```

Comment:
To go with a similar wording to `cdk diff`, perhaps:
> Compares the specified stack with the deployed stack or a local template file
```suggestion
	diffFlagDescription = "Compares the generated CloudFormation template to the deployed stack."
```